test/integration/full-content/full-content.test.js

```diff
@@ -203,13 +203,26 @@ describe( 'full post content fixture', () => {
 			try {
 				expect( serializedActual ).toEqual( serializedExpected );
 			} catch ( err ) {
-				throw new Error(
-					format(
-						"File '%s' does not match expected value:\n\n%s",
-						serializedHTMLFileName,
-						err.message
-					)
-				);
+				if (
+					serialize( blocksActual ) ===
+					serialize( parse( serializedExpected ) )
+				) {
+					throw new Error(
+						format(
+							"File '%s' does not match expected value (however, the block re-serializes identically):\n\n%s",
```

Comment:
Should we also add some detail about the potential fix, eg. `File '%s' does not match expected value (however, the block re-serializes identically so you may need to run 'npm run fixtures:regenerate')` ?

Reply:
Great idea! I'll update the message, cheers.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/39039/commits/c46cbbf2a38caa9a4bd095c5bfcf4b577b0f18e9 — that reads better, and I think that'll hopefully make it a bit easier for the next person who encounters the error 👍